CMakeLists.txt

```diff
@@ -598,15 +601,27 @@ endforeach()
 string(REPLACE "\\" "/" ELASTIX_ITK_DIR ${ITK_DIR})
 
 # Create the Config.cmake file for the build tree:
+set(ElastixConfig_TREE "build")
+set(ElasstixConfig_CODE "")
```

Comment:
Please mind the typo: "ElasstixConfig_CODE" should only have one `s`!

Reply:
done